src/actors/symbolication.rs

```diff
@@ -359,15 +393,17 @@ impl SourceLookup {
 
         for stacktrace in stacktraces {
             for frame in &stacktrace.frames {
-                if let Some(i) = self.get_object_index_by_addr(frame.raw.instruction_addr.0) {
+                if let Some(i) =
+                    self.get_object_index_by_addr(frame.raw.instruction_addr.0, frame.raw.addr_mode)
```

Comment:
It would be great to introduce a struct that contains both, and potentially carries helper functions.